themes-default/slim/views/vue-components/anidbReleaseGroupUi.mako

```diff
@@ -0,0 +1,177 @@
+<style scoped>
```

Comment:
Used scoped, don't know if thats a problem

Reply:
No browser actually supports that according to https://caniuse.com/#feat=style-scoped